server/services/v1/service.go

```diff
@@ -41,7 +41,6 @@ func GetRegisteredServicesRealtime(kvStore kv.KeyValueStore, searchStore search.
 	var v1Services []Service
 	v1Services = append(v1Services, newRealtimeService(kvStore, searchStore, tenantMgr, txMgr))
 	v1Services = append(v1Services, newHealthService(txMgr))
-	v1Services = append(v1Services, newObservabilityService(tenantMgr))
```

Comment:
why is this removed?

Reply:
I don't think we need it for real-time. But probably for console we need it?